packages/shared/lib/core/wallet/utils/isImplicitAccountOutput.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    OutputData,
+    OutputType,
+    CommonOutput,
+    UnlockConditionType,
+    AddressUnlockCondition,
+    AddressType,
+} from '@iota/sdk'
+
+export function isImplicitAccountOutput(outputData: OutputData): boolean {
+    const output = outputData?.output
```

Comment:
for this it is better to go through param `isImplicitAccountOutput(output: CommonOutput):`